python/pyarrow/tests/test_serialization.py

```diff
@@ -650,3 +650,16 @@ def loads2(serialized_obj):
     serialized = pa.serialize(test_object, context=context).to_buffer()
     deserialized = pa.deserialize(serialized.to_pybytes(), context=context)
     assert deserialized == b'custom serialization 2'
+
+
+def test_repeated_serialization():
+    # ARROW-1996
+    for u, v in zip(PRIMITIVE_OBJECTS, PRIMITIVE_OBJECTS[1:]):
```

Comment:
This could be parametrized so each one appears as an individual test and is therefore easier to visually inspect if it fails.

Reply:
The rest of this test file doesn't use parametrization (I'm also not a fan of test parametrization :-)).

Reply:
Fair enough :)